src/components/section-editorial/section-editorial.js

```diff
@@ -88,6 +91,10 @@ const SectionEditorial = ({
     id = undefined
   }
 
+  if (!componentViewerData) {
+    componentViewerData=null
+  }
+
```

Comment:
This is odd. I think the Component below should handle when `componentViewerData` is falsy by itself.